src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingInvocationReasons_Constants.cs

```diff
@@ -13,37 +13,47 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.SolutionCrawler
         public static readonly UnitTestingInvocationReasons DocumentAdded =
             new(
                 ImmutableHashSet.Create<string>(
+#if false // Not used in unit testing crawling
                                     UnitTestingPredefinedInvocationReasons.DocumentAdded,
                                     UnitTestingPredefinedInvocationReasons.SyntaxChanged,
+#endif
                                     UnitTestingPredefinedInvocationReasons.SemanticChanged));
 
         public static readonly UnitTestingInvocationReasons DocumentRemoved =
             new(
                 ImmutableHashSet.Create<string>(
+#if false // Not used in unit testing crawling
                                     UnitTestingPredefinedInvocationReasons.DocumentRemoved,
                                     UnitTestingPredefinedInvocationReasons.SyntaxChanged,
+#endif
                                     UnitTestingPredefinedInvocationReasons.SemanticChanged,
                                     UnitTestingPredefinedInvocationReasons.HighPriority));
 
+#if false // Not used in unit testing crawling
         public static readonly UnitTestingInvocationReasons ProjectParseOptionChanged =
```

Comment:
Interesting. What is this? Our assumption has been that any project-level changes that impact Roslyn (compilation / analysis) would trigger ProjectconfigurationChanged below. Is this an invalid assumption?

Reply:
yup.   First, to be clear, if parse options change, we would emit ProjectConfigChanged.  However, we'd *also* give hte more specific info that the parse options changed.

However, since you don't use the latter, this was removed and made simpler.  You'll still get the ProjectConfigChanged notification though.

Reply:
Got it - glad to know there will be no behavior change!